### PR TITLE
[PVR][Estuary] Add GUI for Radio Data System (RDS).

### PR DESCRIPTION
**Pull request by ksooo**
Kodi VideoPlayer supports Radio Data RDS (RDS) for a long time, btw, but somehow we forgot to add Estuary support...

The PR completes and fixes RDS core and Estuary support. 

**Before:**
* a rudimentary RDS info dialog which was not even directly accessible from Estuary:

screenshot000

**After:**
* completely reworked RDS info dialog, accessible from PVR radio OSD
* player info extended to display "rds" label if an RDS stream is available for the currently playing channel.
* bonus: pvr radio OSD now has buttons for "channels" and "channel guide" (as Live TV OSD has)

screenshot002
screenshot001
screenshot018

For sure there is room for functional improvements, but this is for another day. ;-)

@ronie are the skin changes okay?
@Jalle19 mind doing a code review?
@da-anda fyi

### CODE REVIEW

File: addons/skin.estuary/xml/MusicOSD.xml [hunk 1]
```diff
@@ -112,6 +112,30 @@
 				<ondown>noop</ondown>
 				<onleft>606</onleft>
 				<onright>600</onright>
+				<control type="radiobutton" id="62040">
+					<include content="OSDButton">
+						<param name="texture" value="osd/fullscreen/buttons/channels.png"/>
+					</include>
+					<onclick>Dialog.Close(MusicOSD)</onclick>
+					<onclick>ActivateWindow(PVROSDChannels)</onclick>
+					<visible>VideoPlayer.Content(LiveTV)</visible>
```
Comment (ronie) [inline addons/skin.estuary/xml/MusicOSD.xml]: referring to VideoPlayer in the MusicOSD seems a bit odd to me?
or is there any usecase i'm unaware of?

File: addons/skin.estuary/xml/MusicOSD.xml [hunk 2]
```diff
@@ -112,6 +112,30 @@
...
+				</control>
+				<control type="radiobutton" id="62041">
+					<include content="OSDButton">
+						<param name="texture" value="osd/fullscreen/buttons/guide.png"/>
+					</include>
+					<onclick>Dialog.Close(MusicOSD)</onclick>
+					<onclick>ActivateWindow(PVRChannelGuide)</onclick>
+					<visible>VideoPlayer.Content(LiveTV) + VideoPlayer.HasEPG</visible>
```
Comment (ronie) [inline addons/skin.estuary/xml/MusicOSD.xml]: same..

File: addons/skin.estuary/xml/Variables.xml [hunk 3]
```diff
@@ -372,6 +375,12 @@
 		<value condition="!String.IsEmpty(Window.Property(IsRadio))">$LOCALIZE[19199] - $LOCALIZE[19024]</value>
 		<value>$LOCALIZE[19199] - $LOCALIZE[19023]</value>
 	</variable>
+	<variable name="RDSInfoSubTitle">
+		<value condition="String.IsEmpty(MusicPlayer.ChannelName)">$INFO[Player.Title]</value>
+		<value condition="String.IsEmpty(Player.Title)">$INFO[MusicPlayer.ChannelName]</value>
+		<value condition="String.IsEmpty(MusicPlayer.ChannelName) + String.IsEmpty(Player.Title)"></value>
+		<value>$INFO[MusicPlayer.ChannelName] - $INFO[Player.Title]</value>
```
Comment (ronie) [inline addons/skin.estuary/xml/Variables.xml]: the three lines above can be merged into one:
`<value>$INFO[MusicPlayer.ChannelName]$INFO[Player.Title, - ]</value>`

File: addons/skin.estuary/xml/MusicOSD.xml (hunk 1 above)
Comment (ksooo) [reply addons/skin.estuary/xml/MusicOSD.xml]: c/p error. will change.

File: addons/skin.estuary/xml/MusicOSD.xml (hunk 2 above)
Comment (ksooo) [reply addons/skin.estuary/xml/MusicOSD.xml]: c/p error. will change.

File: addons/skin.estuary/xml/Variables.xml (hunk 3 above)
Comment (ksooo) [reply addons/skin.estuary/xml/Variables.xml]: Hehe, ofc. Thanks.

**Review by Jalle19 — APPROVED** [review]

**Comment by da-anda** [comment]
awesome work. Thanks so much for spending time on this

**Comment by ksooo** [comment]
Jenkins error is unrelated.

**Comment by ksooo** [comment]
Thanks for the kind words, Alwin.

**Pull request merged**